mappings/net/minecraft/predicate/entity/EntityPredicate.mapping

```diff
@@ -6,12 +6,38 @@ CLASS net/minecraft/unmapped/C_kvkvpjlm net/minecraft/predicate/entity/EntityPre
 	FIELD f_ifyilwup passenger Lnet/minecraft/unmapped/C_kvkvpjlm;
 	FIELD f_lmuipoqi steppingOn Lnet/minecraft/unmapped/C_ypvebgnl;
 	FIELD f_mxeamups team Ljava/lang/String;
+	FIELD f_niwouezd typeSpecific Lnet/minecraft/unmapped/C_ljtxhtrw;
```

Comment:
What is this? It's grammatically strange.

Reply:
a type specific predicate is specific to an entity type and/or variant
it's a little bit hard to name

Reply:
Oh, and I guess "predicate" is implied for all these fields, ok.